src/test/java/uk/gov/hmcts/reform/sscs/service/VenueServiceTest.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.reform.sscs.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import java.util.Map;
+import java.util.Optional;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.sscs.model.VenueDetails;
+
+
+@RunWith(MockitoJUnitRunner.class)
+public class VenueServiceTest {
+
+    public static final String PROCESSING_VENUE_1 = "test_place";
+    public static final String PROCESSING_VENUE_2 = "test_other_place";
+
+    @InjectMocks
+    private VenueService venueService;
```

Comment:
Very minor / Optional: Usually the unit under test is the last field in the class